distribution/openrct2.d.ts

```diff
@@ -1214,7 +1225,7 @@ declare global {
         readonly remainingDistance: number;
 
         /**
-         * List of peep IDs ordered by seat.
+         * List of guest IDs ordered by seat.
          */
         peeps: Array<number | null>;
```

Comment:
This should be renamed to `guests` too. `peeps` should also still be around for now, but marked as deprecated.